packages/components/inputs/localized-multiline-text-input/src/localized-multiline-text-input.spec.js

```diff
@@ -272,3 +283,43 @@ describe('when the error is on the selected language', () => {
     expect(getByText('Some error')).toBeInTheDocument();
   });
 });
+
+describe('when the additionalInfo exists on the selected locale should display', () => {
+  it('when given localized data', () => {
```

Comment:
Please, avoid repeating test names 🙏 
```suggestion
  it('when given localized data as string', () => {
```